### DISCUSSION TITLE
Batch on Windows

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#2071] [Help]
Hello,
I want a Windows batch script which extract all tiff images of the current directory to jpeg files in the current dircetory please.
With imagemagick of course
Thank you
Kinds regards

**Comment by SylvainArd** [comment]
I precise that my tiff files are multipage

**Reply by fmw42** [reply]
Please always provide your ImageMagick version as syntax may vary.

**Comment by SylvainArd** [comment]
If you have the Ubuntu bash version it would be great !

**Comment by snibgo** [comment]
magick mogrify -format jpeg *.tiff
